### ISSUE TITLE
Flash meeting log sections in focus when directed

### ISSUE THREAD

**Issue by t0xic0der** [fedora-infra/mote#177]
Example - https://css-tricks.com/on-target/

**Comment by t0xic0der** [comment]
@chinmaym07, could you please provide an ETA for this?

**Comment by chinmaym07** [comment]
Hey @t0xic0der . I will take atmost 2 days to complete this issue . Will it work ?

**Comment by t0xic0der** [comment]
2 days is fine. Please limit your PR to what is been requested in the issue ticket.

**Comment by t0xic0der** [comment]
The recent meetings would be inaccessible as the archive was taken some time back. Try opening meetings from the other buttons.

**Comment by chinmaym07** [comment]
I tried all the links which are there in `Last day` & `Last week` tabs but all are giving the same response 404 Not Found . also there are no channels in `explore` ..

**Comment by t0xic0der** [comment]
Please take the instructions provided here https://github.com/t0xic0der/fragment for a spin and let me know if it works.

**Comment by chinmaym07** [comment]
Hey I tried to access this file in drive . But I don't have access to it . I have requested you for access .
https://drive.google.com/file/d/1iOKEgyHMtpnCwsuZa5ufT8gI9-ntuOVc/view?usp=sharing

**Comment by t0xic0der** [comment]
I've shared it with you. :)

**Comment by chinmaym07** [comment]
Hey I have downloaded & extracted the zip file but 404 Not found issue is still there . I tried to log the meetcont fetched using `
fetch_meeting_content` method defined in `modules/call.py`. There is some exception due to which I am getting `(false,'')`.

![image](https://user-images.githubusercontent.com/44670961/135708661-4178fc9c-c056-48e3-921a-1b89673dd5f1.png)

**Comment by t0xic0der** [comment]
Are you sure you extracted the contents in the right location?

**Comment by chinmaym07** [comment]
Yes I think I did ..  
![image](https://user-images.githubusercontent.com/44670961/135753704-d08d3a0a-332a-4bee-baf5-bf5d9462fcb2.png)

**Comment by t0xic0der** [comment]
So the meetings from the recent menu would result in a 404 and that is a
normal behavior as the meeting archive had been taken some time ago. Please
check if you are getting 404 while opening up meetings from either explore
or search menu.

**Comment by chinmaym07** [comment]
Yes that's correct as there are no folder's for the recent meetings due to which it is resulting in 404 which is fine .. therefore I tried to access the last meeting in any folder but that too resulted in 404 .

**Comment by t0xic0der** [comment]
Are meetings getting listed under Explore menu?

**Comment by chinmaym07** [comment]
No there are no meetings available in Explore menu.

![image](https://user-images.githubusercontent.com/44670961/135757404-9d572c98-1554-4bc9-8ebb-86cbf1b8a5f7.png)

**Comment by chinmaym07** [comment]
Hey I have tried something . Will this work ?

https://user-images.githubusercontent.com/44670961/135765155-860b56a6-d09d-421b-ab66-7d3bb4944a9b.mp4

**Comment by t0xic0der** [comment]
Hello,

Fantastic work. This will work.

Could you please change the colour to a lighter gradient of #008080?

Thanks and regards,
Akashdeep Dhar
